src/features/projectsV2/ProjectsMap/ProjectPopup/index.tsx

```diff
@@ -28,6 +35,7 @@ const ProjectPopup = ({ project, handlePopupLeave, visitProject }: Props) => {
         <ProjectSnippet
           project={project.properties}
           showTooltipPopups={false}
+          selectedMode={selectedMode}
         />
```

Comment:
Why do we need to pass `selectedMode` down here?

Reply:
This prop has been removed